docs/src/main/sphinx/release/release-471.md

```diff
@@ -0,0 +1,50 @@
+# Release 471 (19 Feb 2025)
+
+## General
+
+* Include split count and total split distribution time in the `EXPLAIN ANALYZE`
+  output. ({issue}`25028`)
+* Improve performance for queries using top N operations. ({issue}`24937`)
+* Add further packages for use with Python user-defined functions. ({issue}`25058`)
+* Improve performance for Python user-defined functions. ({issue}`25058`)
+
+## Security
+
+* Fix initialization of the Ranger access control system with additional
+  configuration files. ({issue}`24887`)
+
+## Delta Lake connector
+
+* Add support for reading `variant` type. ({issue}`22309`)
+* Add support for local file system. ({issue}`25006`)
```

Comment:
@wendigo @electrum .. will change these to link to docs once merged .. but also .. its for all four connectors right?